main.py

```diff
@@ -0,0 +1,113 @@
+import sqlite3
+import random
+import threading
+import os
+import dbActions
+import Settings
+
+
+def finish_game(name, level):
+    """
+    inserts player's name and score, and finishes the game.
+    :param name: player's name
+    :param level: player's score
+    """
+    con = raise_database()
+    dbActions.insert_new_score(name, level, con, Settings.db_table_name)
+    print "game over"
+    os._exit(1)
+
+
+def calculate_result(first_num, second_num, operation):
+    """
+    calculates equation result
+    :param first_num
+    :param second_num
+    :param operation
+    :return: math result
+    """
+    if operation == '/':
+        return first_num / second_num
+    elif operation == '*':
+        return first_num * second_num
+    elif operation == '+':
+        return first_num + second_num
+    elif operation == '-':
+        return first_num - second_num
+
+
+def check_no_zero(operation, second_num, level):
+    """
+    ensures that we would not divide by zero
+    :param operation
+    :param second_num
+    :param level: in case we need to generate new number
+    :return: original number or a new one
+    """
+    if operation == '/':
+        while second_num == 0:
+            second_num = get_random_num(level)
+    return second_num
+
+
+def lottery_numbers(level):
+    """
+    generates 2 numbers and sing and calculates result of the equation
+    :param level: by the level it will be generated
+    :return: equation to print to the user and its result
+    """
+    first_num = get_random_num(level)
+    second_num = get_random_num(level)
+    mathematical_sign = ['+', '-', '*', '/']
+    operation = random.choice(mathematical_sign)
+    second_num = check_no_zero(operation, second_num, level)
+    equation = str(first_num),operation.replace("'", ""),str(second_num)
+    result = calculate_result(first_num, second_num, operation)
+    return equation, result
+
+
+def get_random_num(level):
```

Comment:
Nice!
I would make the levels a bit easier, 'cause right now each level is a huge jump in the difficulty level.